media-video/pipewire/pipewire-0.3.24.ebuild

```diff
@@ -0,0 +1,228 @@
+		-Dspa-plugins=enabled
+		-Dalsa=enabled # Allows using kernel ALSA for sound I/O (-Dmedia-session depends on this)
+		-Daudiomixer=enabled # Matches upstream
+		-Daudioconvert=enabled # Matches upstream
+		$(meson_feature bluetooth bluez5)
+		$(meson_feature bluetooth bluez5-backend-hsp-native)
+		$(meson_feature bluetooth bluez5-backend-hfp-native)
+		$(meson_feature bluetooth bluez5-backend-ofono)
+		$(meson_feature bluetooth bluez5-backend-hsphfpd)
+		$(meson_feature aac bluez5-codec-aac)
+		$(meson_feature aptx bluez5-codec-aptx)
+		$(meson_feature ldac bluez5-codec-ldac)
+		-Dcontrol=enabled # Matches upstream
+		-Daudiotestsrc=enabled # Matches upstream
+		$(meson_feature extra ffmpeg) # Disabled by upstream
+		$(meson_feature jack) # Allows PW to act as a JACK client
+		-Dsupport=enabled # Miscellaneous/common plugins, such as null sink
+		-Devl=disabled # Matches upstream
+		$(meson_feature debug test) # fakesink and fakesource plugins
+		$(meson_feature v4l v4l2)
+		-Dlibcamera=disabled # libcamera is not in Portage
+		-Dvideoconvert=enabled # Matches upstream
+		-Dvideotestsrc=enabled # Matches upstream
+		-Dvolume=enabled # Matches upstream
+		$(meson_feature debug vulkan) # Uses pre-compiled Vulkan compute shader to provide a CGI video source (disabled by upstream)
+		$(meson_feature extra pw-cat)
+		-Dudev=enabled
+#		-Dudevrulesdir="" # Directory for udev rules (defaults to /lib/udev/rules.d)
+#		-Dsystemd-user-unit-dir="" # Directory for user systemd units (defaults to pkg-config systemd --variable=systemduserunitpath ; prefix awareness expected but not verified)
+		$(meson_feature debug sdl2) # Enables SDL2 dependent code (currently only examples when -Dinstalled_tests=enabled)
+		$(meson_feature extra sndfile) # Enables libsndfile dependent code (currently only pw-cat)
+		# Gentoo added options
+	)
+	meson_src_configure
+}
+
+src_compile() {
+	meson_src_compile
+
+	einfo "Generating ${limitsdfile}"
+	cat > ${limitsdfile} <<- EOF || die
+		# Start of ${limitsdfile} from ${P}
+
+		1000:60000	-	memlock 256
+
+		# End of ${limitsdfile} from ${P}
+	EOF
+}
+
+src_install() {
+	meson_src_install
+
+	insinto /etc/security/limits.d/
+	doins ${limitsdfile}
+
+	if use pipewire-alsa; then
+		keepdir /etc/alsa/conf.d
+
+		dosym8 -r /usr/share/alsa/alsa.conf.d/50-pipewire.conf /etc/alsa/conf.d/50-pipewire.conf
+		dosym8 -r /usr/share/alsa/alsa.conf.d/99-pipewire-default.conf /etc/alsa/conf.d/99-pipewire-default.conf
+	fi
+}
+
+pkg_postinst() {
+	elog "It's recommended to install and enable sys-auth/rtkit daemon to allow"
+	elog "PipeWire to obtain restricted realtime capabilities at runtime."
+	elog
+		if use bluetooth; then
+		elog "For oFono support install and enable net-misc/ofono daemon, and"
+		elog "pipewire-media-session should automatically contact it at runtime."
+		elog
+	fi
+	if !use pipewire-alsa; then
+		elog "PulseAudio's ALSA plugin is being used to integrate ALSA applications"
```

Comment:
Can we make this a bit shorter? Direct advice or link to the wiki for a discussion on it, rather than too even-handed. I get the point being made here, it's just that users will end up glossing over it and it'll fill a lot of lines in the terminal.

Reply:
I think I'm happy with the rest otherwise.

Reply:
That's a fair and good point - I probably did overreact, having spent an entire work day's amount of time over two day period on debugging why all of a sudden the reliable PulseAudio plugin is broken with PipeWire even with known good package versions, USE flag combos and kernels.